PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/CentralRuleConfigLoader.java

```diff
@@ -0,0 +1,233 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.azure.json.JsonReader;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonProviders;
+import com.google.gson.JsonParseException;
```

Comment:
is this the correct `JsonParseException`?

Reply:
discussed offline -- JsonParseException is never an exception with azure-json so safe to take out from the logger

Reply:
catching IOException should be enough